### PR TITLE
Update use cases

### PR DESCRIPTION
**Pull request by BirgitBader**

### CODE REVIEW

File: markdown_stoplight/use-cases.md [hunk 1]
```diff
@@ -1,128 +1,157 @@
...
+That’s great!
+Here are some use cases we can think of:
 
-### Integrate a Payment Solution
+### Payment solution
 
-Enable merchants to offer more payment options by integrating your own payment solution. Create your own payment method so that merchants can later activate it for their shops. This way, a merchant's customers will find a more diverse set of possibilities to pay for their purchase.</br>
+Enable merchants to offer more payment options by integrating your own payment solution.
+Create your own payment method so that merchants can later activate it for their shops.
+This way, a merchant's customers will find a more diverse set of possibilities to pay for their purchase.</br>
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md]: Nothing new, but: a merchant's customers --> Either make it plural or singular

File: markdown_stoplight/use-cases.md [hunk 2]
```diff
@@ -1,128 +1,157 @@
...
 A few of the endpoints to achieve this:
 
 | HTTP Verb | Endpoint
 |---|---
-|`GET`|[/orders](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list)
-|`GET`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
-|`GET`|[/order/{orderId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
-|`GET`|[/orders/{orderId}/processes/shippings](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-processes) |`GET`|[/orders/{orderId}/processes/shippings/{shippingProcessId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-process)
+|`GET`|[/orders](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list)
+|`GET`|[/products/{productId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:42]: I get a 404 for this link... not sure which endpoint is meant here...

File: markdown_stoplight/use-cases.md [hunk 3]
```diff
@@ -1,128 +1,157 @@
...
+|`GET`|[/order/{orderId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
+|`GET`|[/orders/{orderId}/processes/shippings](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-processes) |`GET`|[/orders/{orderId}/processes/shippings/{shippingProcessId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-process)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:44]: In the endpoint itself the ids are written like this: /orders/{order-id}/processes/shippings/{shipping-process-id}

File: markdown_stoplight/use-cases.md [hunk 4]
```diff
@@ -1,128 +1,157 @@
...
 A few of the endpoints to achieve this:
 
 | HTTP Verb | Endpoint
 |---|---
-|`GET`|[/orders](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list) 
-|`GET`|[/order/{orderId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
-|`GET`|[/orders/{orderId}/processes](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-order-processes)
-|`GET`|[/orders/{orderId}/order-note](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-note-update)
+|`GET`|[/orders](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list) 
+|`GET`|[/order/{orderId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:57]: Same issue with the id here and for the next two endpoints..

File: markdown_stoplight/use-cases.md [hunk 5]
```diff
@@ -1,128 +1,157 @@
...
 |---|---
-|`GET`|[/products](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/products-list)
-|`PATCH`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-patch-json)
-|`GET`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
-|`GET`|[/categories](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/categories/categories-list)
-|`POST`|[/categories](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/categories/category-create)
-|`PUT`|[/categories/{categoryId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/categories/category-put)
+|`GET`|[/products](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/products-list)
+|`PATCH`|[/products/{productId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-patch-json)
+|`GET`|[/products/{productId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:74]: Again the 404 link that appeared already in the Shipping section

File: markdown_stoplight/use-cases.md [hunk 6]
```diff
@@ -1,128 +1,157 @@
...
+|`GET`|[/categories](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/categories/categories-list)
+|`POST`|[/categories](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/categories/category-create)
+|`PUT`|[/categories/{categoryId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/categories/category-put)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:77]: The real endpoint is named differently: /categories/{id}

File: markdown_stoplight/use-cases.md [hunk 7]
```diff
@@ -1,128 +1,157 @@
...
 
 | HTTP Verb | Endpoint
 |---|---
-|`GET`|[/products](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/products-list)
-|`GET`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
-|`GET`|[/products/{productId}/availability](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-availability-get)
-|`PATCH`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-patch-json) 
-|`POST`|[/products/{productId}/availability/adjust-available-stock](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-availability-adjust-available-stock)
+|`GET`|[/products](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/products-list)
+|`GET`|[/products/{productId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:106]: Again the 404 link that appeared already in the Shipping section

File: markdown_stoplight/use-cases.md [hunk 8]
```diff
@@ -1,128 +1,157 @@
...
 | HTTP Verb | Endpoint
 |---|---
-|`GET`|[/products](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/products-list)
-|`GET`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
-|`POST`|[/products](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/products-create) 
-|`DELETE`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-delete)
-|`GET`|[/orders](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list)
\ No newline at end of file
+|`GET`|[/products](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/products-list)
+|`GET`|[/products/{productId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:154]: Again the 404 link that appeared already in the Shipping section

File: markdown_stoplight/use-cases.md [hunk 9]
```diff
@@ -1,128 +1,157 @@
...
+|`POST`|[/products](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/products-create)
```
Comment (cgebken) [inline markdown_stoplight/use-cases.md:155]: Link results in a 404 as the endpoint is not yet available in Stoplight

**Review by cgebken — CHANGES_REQUESTED** [review]

File: markdown_stoplight/use-cases.md (hunk 1 above)
Comment (BirgitBader) [reply markdown_stoplight/use-cases.md]: 👍

File: markdown_stoplight/use-cases.md (hunk 2 above)
Comment (BirgitBader) [reply markdown_stoplight/use-cases.md:42]: Me, too. It think, final verification needs to wait until Team Black has done the bug fixes.

File: markdown_stoplight/use-cases.md (hunk 3 above)
Comment (BirgitBader) [reply markdown_stoplight/use-cases.md:44]: Know issue. Has been reported to the team already.

File: markdown_stoplight/use-cases.md (hunk 4 above)
Comment (BirgitBader) [reply markdown_stoplight/use-cases.md:57]: See above comment

**Pull request merged**